client/src/Components/Dashboard/Dashboard.js

```diff
@@ -78,7 +79,7 @@ class Dashboard extends Component {
                 />
                 <Route
                   path="/dashboard/employees"
-                  render={props => <EmployeeList {...props} />}
+                  render={() => <Employees />}
```

Comment:
Be sure to spread props over the render so we have access to them in the page.

Reply:
I'm no longer going to be using props like this.